contributor/reference/stable-release.rst

```diff
@@ -0,0 +1,76 @@
+Stable release process
+======================
+
+Key Steps at Each Stage
+-----------------------
+
+Rocks
++++++
+
+Currently, the rocks are published directly to the registry when a modification is made.
+The Rocks mono-repository is maintains a stable branch from which new rocks are built and published.
```

Comment:
I'd recommend using `monorepo` as it's a popular and known industry term.
[* nit *]: The `Rocks` monorepo maintains... (remove "is").